docs/group-conversations.md

```
@@ -228,7 +238,7 @@ All discussion styles use the same rendering pattern in Slack:
 | **PEER_REVIEW** | Agents post → Critiques thread under targets → Revisions thread under own → Synthesis |
 | **DEVIL_ADVOCATE** | Agent posts → Challenger threads challenges → Agent threads defense → Synthesis |
 | **DEBATE** | PRO agent posts → CON agent posts → Rebuttals thread under opponents → Judge synthesizes |
-| **DELPHI** | Round 1 agents post → Round 2 agents post (convergence) → Synthesis |
+| **DELPHI** | Round 1 agents post → Round 2 agents post (convergence) → Synthesis |\r\n| **TASK_FORCE** | Moderator posts plan → Agents post task results → Verifiers thread under targets → Synthesis |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)